src/components/WorkOrders/MobileWorkingWorkOrdersView.js

```diff
@@ -95,13 +102,15 @@ const MobileWorkingWorkOrdersView = () => {
               <ol className="lbh-list mobile-working-work-order-list">
                 {renderWorkOrderListItems(
                   currentUser.isOneJobAtATime
-                    ? inProgressWorkOrders
-                        .sort((a, b) =>
-                          a.appointment.assignedStart.localeCompare(
-                            b.appointment.assignedStart
+                    ? startedWorkOrders?.length
+                      ? startedWorkOrders
+                      : inProgressWorkOrders
+                          .sort((a, b) =>
+                            a.appointment.assignedStart.localeCompare(
+                              b.appointment.assignedStart
+                            )
                           )
-                        )
-                        .slice(0, 1)
+                          .slice(0, 1)
                     : inProgressWorkOrders
```

Comment:
I think this would be better as a function, with explicit comments. This is going to be very important logic, why not make it as clear to understand as the logic you mentioned above? Perhaps a bit over the top, maybe. But who knows who might be looking at this code.

Suggestion:
```js
  const sortWorkOrderItems = () => {
    // The operative is NOT an OJAAT operative
    if (!currentUser.isOneJobAtATime) return inProgressWorkOrders;

    // If the operative has started a work order
    if (startedWorkOrders?.any()) return startedWorkOrders

    // Return the next unstarted work order 
    return inProgressWorkOrders.sort((a, b) => {
      return a.appointment.assignedStart.localeCompare(
        b.appointment.assignedStart
      )
    })
    .slice(0, 1)  
  }
```

Reply:
Refactored out to a method